napari/_qt/qthreading.py

```diff
@@ -574,6 +587,26 @@ def long_function(duration):
                     )
                 getattr(worker, key).connect(v)
 
+    if _progress is not None:
```

Comment:
providing an empty dict (which is Falsy) feels a little funny as a way indicate "yes, please give me a progress bar...".  could this also support `progress=True`?